packages/runtime/src/state/modules/breakpoints.ts

```diff
@@ -144,56 +131,3 @@ function validateBreakpointsInput(input: BreakpointsInput) {
     }
   })
 }
-
-export const getBreakpoint = (state: State, breakpointId: Breakpoint['id']): Breakpoint => {
```

Comment:
Moved to [`@makeswift/controls`](https://github.com/makeswift/makeswift/pull/947/files#diff-7953999eb345672c2499ad733f28a8068ab26e84f3c4abf6736ae8363c2eb1ba), see the corresponding comments there.